contracts/ShortNameClaims.sol

```diff
@@ -32,24 +32,48 @@ contract ShortNameClaims is Ownable {
     using BytesUtils for bytes;
     using StringUtils for string;
 
+    enum Phase {
+        OPEN,
+        REVIEW,
+        FINAL
+    }
+
+    enum Status {
+        PENDING,
+        APPROVED,
+        DECLINED,
+        WITHDRAWN
+    }
+
     struct Claim {
         bytes32 labelHash;
         address claimant;
         uint paid;
+        Status status;
```

Comment:
Should this field ever be set anything apart from APPROVED or DECLINED? If not, maybe better to make it boolean status. I had to read the entire code to understand that `Status.WITHDRAWN` is only set on event and never be set on claim.status

Reply:
Yes - initially it's PENDING.